src/kite.js

```diff
@@ -477,6 +479,22 @@ export const Kite = {
               .catch(e => {
                 NotificationsManager.showKiteInstallErrorNotification(e);
               });
+            }
+
+            KiteAPI.canDownloadKite().then(yes => {
+              if (!yes) {
```

Comment:
Do we also need to see if the user has Kite installed already to prevent these notification from showing for existing users?